Wiki:Copter: Add CIRCLE_OPTIONS 3rd bit description in Circle mode doc (#5248)
ArduPilot/ardupilot_wiki: copter/source/docs/circle-mode.rst

When bit 1 is set of the :ref:`CIRCLE_OPTIONS<CIRCLE_OPTIONS>` parameter the Copter will face the direction of travel as it circles, otherwise, the Copter will point its nose at the center of the circle as it orbits.
When bit 2 is set of the :ref:`CIRCLE_OPTIONS<CIRCLE_OPTIONS>` parameter the circle's center position will set upon mode entry at the current location, rather than on the perimeter with the center in front of the Copter at the start.

> When bit 3 is set of the :ref:`CIRCLE_OPTIONS<CIRCLE_OPTIONS>` parameter the mount's ROI aka rate of interest will be set on the circle center causing mount to face at circle center all the time. 

Reviewer: ```suggestion
When bit 3 is set of the :ref:`CIRCLE_OPTIONS<CIRCLE_OPTIONS>` parameter the mount's (if used) ROI aka region of interest will be set on the circle center causing mount to face the circle's center all the times. 
```

Author: @Hwurzburg . Sorry . Don't know how i can make such a silly mistake 'Rate' instead of region. Thanks for pointing out
